ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -649,6 +646,16 @@ sub write_file {
     chmod 0644, $self->{Htmlfile} unless $self->{Htmlfile} eq '-';
 }
 
+<<<<<<< HEAD
+=======
+#sub compare {
+#    my ($ha, $hb) = @_;
+#    local $Data::Dumper::Terse  = 1;
+#    local $Data::Dumper::Indent = 0;
+#    local $Data::Dumper::Sortkeys = 1;
+#    return (Dumper($ha) eq Dumper($hb)) ? 1 : 0;
+#}
+>>>>>>> b4b3ed877e (Replace _save_pages() with _transform())
```

Comment:
It appears you have committed a conflicted hunk.  **Surely** this is a mistake!

Reply:
> It appears you have committed a conflicted hunk. **Surely** this is a mistake!

This is puzzling.  I don't see this in the code I pushed to github.  Github is saying that the lines in question are "Outdated".

I just rebased the branch on blead.  Can you double-check?

Reply:
@rjbs, I ack-ed the branch for merge-conflict marks and found none.  I also built `perl` at each commit in the branch and ran the tests in `ext/Pod-Html/t/*.t`.  I got no failures.  So unless I hear from you otherwise, I'm going to assume that what you observed poses no barrier to merging this ticket into blead.

Reply:
Your branch includes the commit 47a5a69b5a46d5e567358f2e10594fcca0698b9a which commits merge markers.  A later commit removes them.

Reply:
> Your branch includes the commit [47a5a69](https://github.com/Perl/perl5/commit/47a5a69b5a46d5e567358f2e10594fcca0698b9a) which commits merge markers. A later commit removes them.

Y'know, I took pains in this project to have small(-ish) commits within each pull request -- but I honestly didn't expect anyone to review on a commit-by-commit basis.  This problem should be repaired now; please double-check.  Thanks for the detailed code review.

Reply:
Looking good!

Reviewing on a "whole diff" versus "commit by commit" basis can be a tough choice.  We were talking about it at $dayjob just last week.  I like commit-by-commit for bigger branches that are altering existing code, *generally*, and whole-diff for new code or a couple small commits.  It's never clear, though.  I wrote a few comments while I was reviewing your work's early commits that I had to then go back and delete in later commits when I saw that my concern was answered by for the end of the commit.

Trade-offs all the way down…

Thanks for your work on this!